src/Runner.Listener/JobDispatcher.cs

```diff
@@ -346,10 +346,7 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                 }
 
                 var term = HostContext.GetService<ITerminal>();
-
-                string workflowName = message.Variables["system.workflowFilePath"].Value.Split('/').LastOrDefault();
-                string additionalInfo = string.IsNullOrEmpty(workflowName) ? $"(in repository \"{_runnerSettings.RepoOrOrgName}\")" : $"(workflow \"{workflowName}\" in repository \"{_runnerSettings.RepoOrOrgName}\")";
```

Comment:
Just FYI, the runner `_runnerSettings.RepoOrOrgName` is for the repo/org the runner configured, it's not where the workflow coming from.
Ex: enterprise/org runner.